tests/http_client.rs

```diff
@@ -0,0 +1,57 @@
+#![cfg(all(any(unix, windows), feature = "http-client-json"))]
+
+use fluvio_future::http_client::{self, ResponseExt, StatusCode};
+
```

Comment:
integrated test should not have outside dependency which could be offline or behavior changed.  Please take look at other service testing.  This means crate fake http server so we can control exact behavior to test

Reply:
There is a nice reference unit test in the hub-pkg-server repo https://github.com/infinyon/hub-service/blob/4841b25d4a13c6f2f035d2310c2600c3d1341b8c/crates/hub-pkg-server/src/util.rs#L314

Reply:
Setting up a standalone https server with a root-certificate derived cert is time consuming, can we mark ignore on external sites referenced in this test, and keep the https://infinyon domain sites in for this test.